structure_tests/types.go

```diff
@@ -35,7 +35,8 @@ func (a *arrayFlags) Set(value string) error {
 }
 
 var schemaVersions map[string]VersionHolder = map[string]VersionHolder{
-	"1.0.0": new(VersionHolderv1),
+	"1.0.0": new(VersionHolderv000),
```

Comment:
Should these be 100 and 110?

Reply:
I struggled with this, maybe you have some input. I tried to make it more clear in the code (not through the actual version string) which versions of each test component are being used for each schema version. So in this case, schema version `1.0.0` is using `v0` of each of the components, so I made that struct `v000`. Schema version `1.1.0` is using `v1` of the command tests, and `v0` of the other two, so I made the struct `v100`. It's a little confusing...

Reply:
Yeah I think it would be easier to follow if they matched. I could see wanting to make the variable names shorter and easier to read, but making them v000 and v100 doesn't really help there...